styles/variables-silicon.less

```diff
@@ -121,7 +121,11 @@
 
 // Heading
 // ---------------------------------------
-@agate-heading-padding: 3px 0;
+@agate-heading-height: 60px;
+@agate-heading-margin: 0;
+@agate-heading-padding: 0 (2.5 * @agate-spotlight-outset) 0 @agate-spotlight-outset;
```

Comment:
I would use static values here : 0 30px 0 9px;
With 9px we are closer to the UX requirement of 10px